modules/nw-ingress-sharding-dns.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the folllowing assemblies:
+//
+// * networking/ingress-sharding.adoc
+
+:_content-type: CONCEPT
+[id="nw-ingress-sharding-dns_{context}"]
+= Ingress sharding and DNS
+
+You need an external DNS server to route requests to the desired shard. Because of this, the cluster administrator is responsible for making a separate DNS entry for each router in a project. A router will not forward unknown routes to another router.
+
+Consider the following example:
+
+* Router A lives on host 192.168.0.5 and has routes with `*.foo.com`.
+* Router B lives on host 192.168.1.9 and has routes with `*.example.com`.
+
+Separate DNS entries must resolve `\*.foo.com` to the node hosting Router A and `*.example.com` to the node hosting Router B:
```

Comment:
```suggestion
Separate DNS entries must resolve `*.foo.com` to the node hosting Router A and `*.example.com` to the node hosting Router B:
```

Reply:
The `\` is an asciidoc escape character

Reply:
I guess it should be consistent everywhere.  Either it's on every `*.text.text` or it's on none of them.

Reply:
There is sometimes an error that occurs when trying to use `*` within inline code, ` `` `, but it is not consistent in asciidoc, and it can show up unexpectedly in the generated HTML. So the escape character is not something we apply in a standard way, just when the issue presents itself. It doesn't impact the execution of the copied code so I'm going to leave things as is